docs/manual/java/gettingstarted/GettingStartedMaven.md

```diff
@@ -67,18 +67,18 @@ $ mvn lagom:runAll
 (Services started, press enter to stop and go back to the console...)
 ```
 
-You can verify that the services are indeed up and running by invoking one of its endpoints, e.g:
+You can verify that the services are indeed up and running by invoking one of its endpoints from any HTTP client. For example, on Windows you might use Postman. On Linux, you might use curl:
```

Comment:
You can edit the file on this branch, create a new commit, and push that to your fork (origin). The pull request will update automatically.